common/src/main/java/whocraft/tardis_refined/common/tardis/manager/TardisControlManager.java

```diff
@@ -239,6 +244,9 @@ public void beginFlight() {
     }
 
     public void endFlight() {
+        TardisNavLocation lastKnown = operator.getControlManager().getTargetLocation();
+        TardisEvents.LAND.invoker().onLand(operator, lastKnown.level, lastKnown.position);
```

Comment:
The TARDIS cannot end a flight until it hasn't met the criteria in line 250. This will act as if the TARDIS is confirmed to be landing.

Reply:
I will move the invoker